runners/wikiConsole/templates/createFile.asciidoc

```diff
@@ -0,0 +1,28 @@
+== Create the file <%= fileName; %>
+
+==== Prerequisites
+* Existing folder you want to load the file. (If the folder doesn't exist you can create it from terminal.)
+
+
+.Operating system
+. Windows + 
+Execute the command to create a folder if not already existing `New-Item -ItemType Directory -Force -Path <%= filePath; %>`.
+. Linux and macOS + 
+Execute the command `mkdir -p <%= filePath; %>`. The `-p` flag won't throw an error if the target folder already exists and allows the creation of parent directories.
+
+=== Create <%= fileName; %> from command prompt
+First, you need to open a command prompt in your current workspace. For Windows, you can use PowerShell and on Linux, you can use the Terminal.
+
+.Operating System
+. Windows + 
+In Powershell, we use the command *New-Item* to create a new file<% if(contentPath) { %>, and to insert some content into the created file we use the *Get-Content* command<% } %>.
+More information about the *New-Item* function and the usage can be found https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/new-item?view=powershell-7.1[here]<% if(contentPath) { %> and some information about the *Get-Content* function and the usage can be found https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-content?view=powershell-7.1[here]<% } %>. 
+To create a new empty <%= fileName; %> <% if(contentPath){ %>and insert some data from into it, <%= contentFile; %> <% } %>you can execute the command `New-Item  -Path "<%= filePath; %>" -Name 
+<%= fileName; %> -ItemType "file" <% if(contentPath) { %> -Value $(Get-Content -Path <%= contentPath; %>)`<% } %>. 
+
+. Linux and macOS +
+In the Terminal, we start using the function *cat* to create a new file <% if(contentPath) { %>and also write the content from another file into it <% } %> 
```

Comment:
I think the term 'command' is more common than 'function' in the context of command lines.